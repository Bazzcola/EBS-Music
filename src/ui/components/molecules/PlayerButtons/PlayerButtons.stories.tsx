import React from 'react';
import { PlayerButtons } from 'ui/components/molecules/PlayerButtons/PlayerButtons';

export default {
  title: 'Molecules/PlayerButtons',
  component: PlayerButtons
};

export const PlayerButtons1 = () => {
  return <PlayerButtons />;
};
