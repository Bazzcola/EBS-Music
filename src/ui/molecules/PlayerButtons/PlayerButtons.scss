.btn_box {
    display: flex;
    justify-content: space-between;
    max-width: 150px;
}