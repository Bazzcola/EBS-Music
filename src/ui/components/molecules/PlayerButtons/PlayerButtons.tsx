import React from 'react';
import { PrevButton } from 'ui/components/atoms/PrevButton/PrevButton';
import { NextButton } from 'ui/components/atoms/NextButton/NextButton';
import { PlayButton } from 'ui/components/atoms/PlayButton/PlayButton';
import 'ui/components/molecules/PlayerButtons/PlayerButtons.scss';

export const PlayerButtons = () => {
  return (
    <div className="btn_box">
      <div className="btn_box__item">
        <PrevButton />
      </div>
      <div className="btn_box__item">
        <PlayButton />
      </div>
      <div className="btn_box__item">
        <NextButton />
      </div>
    </div>
  );
};
