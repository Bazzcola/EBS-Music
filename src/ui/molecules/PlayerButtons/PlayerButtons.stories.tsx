import React from 'react';
import { PlayerButtons } from 'ui/molecules/PlayerButtons/PlayerButtons';

export default {
  title: 'Molecules/PlayerButtons',
  component: PlayerButtons
};

export const PlayerButtons1 = () => {
  return <PlayerButtons />;
};
