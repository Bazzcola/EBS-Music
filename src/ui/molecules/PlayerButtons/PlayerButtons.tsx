import React from 'react';
import { PrevButton } from 'ui/atoms/PrevButton/PrevButton';
import { NextButton } from 'ui/atoms/NextButton/NextButton';
import { PlayButton } from 'ui/atoms/PlayButton/PlayButton';
import 'ui/molecules/PlayerButtons/PlayerButtons.scss';

export const PlayerButtons = () => {
  return (
    <div className="btn_box">
      <div className="btn_box__item">
        <PrevButton />
      </div>
      <div className="btn_box__item">
        <PlayButton />
      </div>
      <div className="btn_box__item">
        <NextButton />
      </div>
    </div>
  );
};
