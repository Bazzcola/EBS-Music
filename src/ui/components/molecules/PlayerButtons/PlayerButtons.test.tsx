import React from 'react';
import { render } from '@testing-library/react';
import { PlayerButtons } from './PlayerButtons';

it('renders PlayerButtons correctly', () => {
  const { asFragment } = render(<PlayerButtons />);
  expect(asFragment()).toMatchSnapshot();
});
