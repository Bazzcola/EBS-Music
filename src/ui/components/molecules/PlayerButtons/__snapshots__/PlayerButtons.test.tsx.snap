// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PlayerButtons correctly 1`] = `
<DocumentFragment>
  <div
    class="btn_box"
  >
    <div
      class="btn_box__item"
    >
      <button
        class="prev_button "
      >
        <svg
          class="svg_prevbtn"
          viewBox="0 0 12 12"
        >
          <path
            d="M2 5.397V.5a.5.5 0 1 0-1 0v11a.5.5 0 1 0 1 0V6.597L11 12V0L2 5.397z"
          />
        </svg>
      </button>
    </div>
    <div
      class="btn_box__item"
    >
      <button
        class="play_button"
      >
        <svg
          class="svg_playbtn"
          viewBox="0 0 12 12"
        >
          <path
            d="M2.5.5v11l9-5.5z"
          />
        </svg>
      </button>
    </div>
    <div
      class="btn_box__item"
    >
      <button
        class="next_button "
      >
        <svg
          class="svg_nextbtn"
          viewBox="0 0 12 12"
        >
          <path
            d="M11 .5v11c0 .276-.232.5-.5.5a.503.503 0 0 1-.5-.5V7.08L1.562 11.95A.375.375 0 0 1 1 11.624V.376c0-.29.312-.47.562-.325L10 4.919V.5c0-.276.232-.5.5-.5.276 0 .5.229.5.5z"
          />
        </svg>
      </button>
    </div>
  </div>
</DocumentFragment>
`;
